sbol_utilities/build_planning.py

```diff
@@ -0,0 +1,142 @@
+import sbol3
+import tyto
+
+from sbol_utilities.component import get_subcomponents, get_subcomponents_by_identity
+
+# TODO: Change SBOL_ASSEMBLY_PLAN and sbol3.SBOL_DESIGN to tyto calls after resolution of 
+SBOL_ASSEMBLY_PLAN = 'http://sbols.org/v3#assemblyPlan'
+ASSEMBLY_TYPES = {sbol3.SBOL_DESIGN, SBOL_ASSEMBLY_PLAN}
+
+
+def validate_part_in_backbone(pib: sbol3.Component) -> bool:
+    """Check if a Component represents a part in backbone
```

Comment:
Comments shouldn't assume the user knows SEP 055 by heart. Add explanation of what it's checking for and why.

Reply:
Fixed by 87d053f